common/landed_titles/wc_religious_titles.txt

```diff
@@ -410,8 +415,7 @@ d_tidemother = {
 	# capital = 1796 # Lesturrom
 # }
 
-### arcane_group
-# cult_of_sunwell
```

Comment:
Why you deleted these comments? It was clearer before. Mentioned the exact name of the religion and the faith.

Reply:
Readded in 13f4d1c0e

Reply:
> Readded in [13f4d1c](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/commit/13f4d1c0edcdd1022fb10521173e9dc6436c8064)

That's not what I meant by the exact game. `arcane_group` and `holy_light` are exact names.

Reply:
back in 56fb20691